community/zsh/build

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh -e
+
+printf "functions='Completion/*comp* Completion/Base/*/* Completion/Linux/*/* Completion/Unix/*/* Completion/X/*/* Completion/Zsh/*/*'\n" >> Src/Zle/complete.mdd
```

Comment:
Something like this is also an option:

```sh
{
    printf "functions='"
    printf "Completion/*comp* "
    printf "Completion/Base/*/* "
    printf "Completion/Zsh/*/* "
    printf "Completion/Linux/*/* "
    printf "Completion/Unix/*/* "
    printf "Completion/X/*/* "
    printf "'\\n"
} >> Src/Zle/complete.mdd
```

This also makes adding/removing entries a little easier. Though a loop may turn out even simpler. ;)

Reply:
Another:

```sh
    printf '%s ' "functions='" "Completion/*comp*"    "Completion/Base/*/*" "Completion/Zsh/*/*" \
                               "Completion/Linux/*/*" "Completion/Unix/*/*" "Completion/X/*/*" "'
" >> Src/Zle/complete.mdd
```

Reply:
Or even:

```sh
    printf '%s ' "functions='" \
                 "Completion/*comp*"    \
                 "Completion/Base/*/*" \
                 "Completion/Zsh/*/*" \
                 "Completion/Linux/*/*" \
                 "Completion/Unix/*/*" \
                 "Completion/X/*/*" "'" >> Src/Zle/complete.mdd
```

Reply:
A loop: 

```sh
printf "functions='Completion/*comp*'\\n" >> Src/Zle/complete.mdd

for comp in Base Zsh Linux Unix X; do
    printf "functions+=' Completion/$comp/*/*'\\n" 
done >> Src/Zle/complete.mdd
```

(Untested)

Reply:
Another loop:

```sh
{
    printf "functions='Completion/*comp* " 

    for comp in Base Zsh Linux Unix X; do
        printf "Completion/$comp/*/* " 
    done >> Src/Zle/complete.mdd

    printf "'"
} >> Src/Zle/complete.mdd
```